tslint.json

```diff
@@ -80,6 +81,16 @@
       true,
       "always"
     ],
+    "template-accessibility-alt-text": true,
```

Comment:
I'd say we should **not** do this for all applications (`ng-bootstrap`, `demo`, `e2e-app`, `ssr-app`), this should be only for the `ng-bootstrap` in `src/tslint.json`

Reply:
> 
> 
> I'd say we should **not** do this for all applications (`ng-bootstrap`, `demo`, `e2e-app`, `ssr-app`), this should be only for the `ng-bootstrap` in `src/tslint.json`

Done.